msteams-platform/messaging-extensions/how-to/search-commands/define-search-command.md

```diff
@@ -124,8 +124,131 @@ For the complete app manifest, see [App manifest schema](~/resources/schema/mani
+
+```json
+{
+    "$schema": "https://developer.microsoft.com/json-schemas/teams/v1.11/MicrosoftTeams.schema.json",
+    "manifestVersion": "1.11",
+    "version": "1.0.0",
+    "id": "%MICROSOFT-APP-ID%",
+    "packageName": "com.example.myapp",
+    "bots": [
+        {
+            "botId": "%MICROSOFT-APP-ID-REGISTERED-WITH-BOT-FRAMEWORK%",
+            "scopes": [
+                    "team",
+                    "personal",
+                    "groupchat"
+                ]
+        }
+    ],
+    "composeExtensions": [
+        {
+            "canUpdateConfiguration": true,
+            "botId": "%MICROSOFT-APP-ID-REGISTERED-WITH-BOT-FRAMEWORK%", // Use the same bot as what is specified in the bots section above
+        }
+    ]
+}
+```
+### Automatic refresh for Adaptive Cards in search based MEs
+
+Enable automatic refresh in Adaptive Cards in messaging extensions to ensure users always see up to date data. To enable, define `userIds` array in either `29:<ID>` or `8:orgid:<AAD ID>` format in the `refresh` property.
+
+**Example**
+
+The following is an example of `userIds` array in the `refresh` property:
+
+```json
+    {
+        "type": "AdaptiveCard",
+        "refresh": {
+            "userIds": [
+                "<8:orgId:<AADID>>",
+                "29:<id>"
+            ],
+            "action": {
+                "type": "Action.Execute",
+                "data": {}
+            }
+        },
+        "body": [
+            {
+                "type": "TextBlock",
+                "text": "Hello World!",
+                "wrap": true
+            }
+        ],
+        "actions": [
+            {
+                "type": "Action.Execute",
+                "data": {},
+                "title": "Hello"
+            }
+        ]
+    }
+```
+> [!NOTE]
+> Automatic refresh is enabled for all users in the group chat or channel with more than or equal to 60 users. For conversations (group chat or channel) with more than 60 users, you can provide a manual refresh button or use the refresh button in the message options menu to get the latest result.
+ 
+```json
+    {
+        "type": "AdaptiveCard",
+        "refresh": {
+            "action": {
+                "type": "Action.Execute",
+                "data": {}
+            }
+        },
+        "body": [
+            {
+                "type": "TextBlock",
+                "text": "Hello World!",
+                "wrap": true
+            }
+        ],
+        "actions": [
+            {
+                "type": "Action.Execute",
+                "data": {},
+                "title": "Hello"
+            }
+        ]
+    }
+```
+
+### Just-In Time (JIT) install
+
+When a user selects a card and sends it in a group chat or channel, a **Just-In Time (JIT)** installation prompt appears. After the user selects the **send** option, the bot installs the card in the background for all the users in the group.
```

Comment:
It would be good to clarify this better.

> In order to support Universal Actions in search based Message extensions, your bot will be added to the conversation where the card (containing `Action.Execute`) is sent by the user.

Reply:
Also, 
> the bot installs the card in the background for all the users in the group.

would be incorrect. I suggest we put

> the bot is added to the group chat/channel in the background